app/views/course_sessions/index.html.erb

```diff
@@ -10,9 +10,10 @@
         <%= course_session.name %>
         <%= link_to fa_icon("trash"), course_session, class: "float-right px-3", method: :delete, data: { confirm: 'Are you sure?' } if @current_user.admin %>
         <%= link_to fa_icon("edit"), edit_course_session_path(course_session), class: "float-right" if @current_user.admin %>
+        <%= content_tag(:span, course_session.status, class: course_session.status_badge) %>
       </div>
       <div class="card-title p-3">
-        <h5><%= "Starts " + course_session.start_to_s %></h5>
+        <h5><%= "Starts " + course_session.start_to_s if course_session.start_date %></h5>
```

Comment:
This should not be needed since you have the same check in the `start_to_s` method.